src/transport/smtp/async_transport.rs

```diff
@@ -132,7 +132,8 @@ where
     }
 }
 
-/// Contains client configuration
+/// Contains client configuration.
+/// Instances of this struct can be created using functions of [`AsyncSmtpTransport`](./struct.AsyncSmtpTransport.html).
```

Comment:
This should be able to use intra-doc links

Reply:
Didn't know that was a thing. 😆 Learned something new today!